diff --git a/debian/rules b/debian/rules
index 693306daa..c7d9bd89a 100755
--- a/debian/rules
+++ b/debian/rules
@@ -134,7 +134,7 @@ perl-test-%-stamp: perl-build-%-stamp
 	dh_auto_test --builddirectory=build-$*
 	touch $@
 
-$(patchlevel): debian/patches/series
+$(patchlevel):
 	$(checkdir)
 	test -f $<	# maintainer sanity check
 	debian/gen-patchlevel -p $(patchprefix) -v $(DEB_VERSION) $< > $@
