diff --git a/debian/libperl5.42.links b/debian/libperl5.42.links
index f574f9fa8..a769f234d 100644
--- a/debian/libperl5.42.links
+++ b/debian/libperl5.42.links
@@ -1,3 +1,5 @@
+usr/lib/${DEB_HOST_MULTIARCH}/perl/${env:fullversion} usr/lib/${DEB_HOST_MULTIARCH}/perl/5.40
+usr/lib/${DEB_HOST_MULTIARCH}/perl5/5.40 usr/lib/${DEB_HOST_MULTIARCH}/perl5/${env:version}
 usr/lib/${DEB_HOST_MULTIARCH}/perl/${env:fullversion}/Config.pm usr/lib/${DEB_HOST_MULTIARCH}/perl/cross-config-${env:fullversion}/Config.pm
 usr/lib/${DEB_HOST_MULTIARCH}/perl/${env:fullversion}/Config_heavy.pl usr/lib/${DEB_HOST_MULTIARCH}/perl/cross-config-${env:fullversion}/Config_heavy.pl
 usr/lib/${DEB_HOST_MULTIARCH}/libperl.so.${env:fullversion} usr/lib/${DEB_HOST_MULTIARCH}/libperl.so.${env:version}
