diff --git a/debian/control b/debian/control
index 1773e8c46..d88852453 100644
--- a/debian/control
+++ b/debian/control
@@ -6,11 +6,11 @@ Uploaders: Dominic Hargreaves <dom@earth.li>
 Standards-Version: 3.9.8
 Homepage: https://dev.perl.org/perl5/
 Rules-Requires-Root: no
+Build-Conflicts: libdb-dev
 Build-Depends: file,
  cpio,
  dpkg-dev (>= 1.22.5),
  libcrypt-dev,
- libdb-dev,
  libgdbm-dev (>= 1.18-3),
  libgdbm-compat-dev,
  netbase <!nocheck>,
@@ -76,6 +76,7 @@ Replaces: perl (<< 5.10.1-12), perl-modules (<< 5.20.1-3),
  libfile-path-perl (<< 2.18),
  libio-socket-ip-perl (<< 0.43),
 Provides: ${perlapi:Provides},
+ perlapi-5.40.1,
  libscalar-list-utils-perl (= 1:1.68.01),
  libxsloader-perl (= 0.32),
  libsocket-perl (= 2.038),
