diff --git a/debian/control b/debian/control
index 1773e8c46..001e49d4c 100644
--- a/debian/control
+++ b/debian/control
@@ -6,11 +6,11 @@ Uploaders: Dominic Hargreaves <dom@earth.li>
 Standards-Version: 3.9.8
 Homepage: https://dev.perl.org/perl5/
 Rules-Requires-Root: no
+Build-Conflicts: libdb-dev
 Build-Depends: file,
  cpio,
  dpkg-dev (>= 1.22.5),
  libcrypt-dev,
- libdb-dev,
  libgdbm-dev (>= 1.18-3),
  libgdbm-compat-dev,
  netbase <!nocheck>,
